tests/api/test_account_permission_group.py

```diff
@@ -127,21 +151,116 @@ def test_permission_group_create_mutation_group_exists(
                 AccountPermissions.MANAGE_USERS.name,
                 AccountPermissions.MANAGE_SERVICE_ACCOUNTS.name,
             ],
+            "users": [graphene.Node.to_global_id("User", staff_user.id)],
         }
     }
     response = staff_api_client.post_graphql(
         query, variables, permissions=(permission_manage_staff,)
     )
     content = get_graphql_content(response)
     data = content["data"]["permissionGroupCreate"]
-    errors = data["bulkAccountErrors"]
+    errors = data["permissionGroupErrors"]
     permission_group_data = data["group"]
 
     assert permission_group_data is None
     assert len(errors) == 1
     assert errors[0]["field"] == "name"
     assert errors[0]["code"] == AccountErrorCode.UNIQUE.name
-    assert errors[0]["index"] is None
+    assert errors[0]["permissions"] is None
+    assert errors[0]["users"] is None
+
+
+def test_permission_group_create_mutation_add_customer_user(
+    staff_user,
+    customer_user,
+    permission_manage_staff,
+    staff_api_client,
+    permission_manage_users,
+    permission_manage_service_accounts,
+):
+    second_customer = User.objects.get(pk=customer_user.pk)
+    second_customer.id = None
+    second_customer.email = "second_customer@test.com"
+    second_customer.save()
+
+    staff_user.user_permissions.add(
+        permission_manage_users, permission_manage_service_accounts
+    )
+    query = PERMISSION_GROUP_CREATE_MUTATION
+
+    user_ids = [
+        graphene.Node.to_global_id("User", user.id)
+        for user in [staff_user, customer_user, second_customer]
+    ]
+    variables = {
+        "input": {
+            "name": "New permission group",
+            "permissions": [
+                AccountPermissions.MANAGE_USERS.name,
+                AccountPermissions.MANAGE_SERVICE_ACCOUNTS.name,
+            ],
+            "users": user_ids,
+        }
+    }
+    response = staff_api_client.post_graphql(
+        query, variables, permissions=(permission_manage_staff,)
+    )
+    content = get_graphql_content(response)
+    data = content["data"]["permissionGroupCreate"]
+    errors = data["permissionGroupErrors"]
+
+    assert errors
+    assert len(errors) == 1
+    assert errors[0]["field"] == "users"
+    assert errors[0]["permissions"] is None
+    assert set(errors[0]["users"]) == set(user_ids[1:])
+    assert errors[0]["code"] == AccountErrorCode.ASSIGN_NON_STAFF_MEMBER.name
+    assert data["group"] is None
```

Comment:
Docstring or comments explaining what we are testing and expecting would be nice